modules/GKB/Release/Utils.pm

```diff
@@ -117,7 +117,7 @@ sub get_dba {
 # C shell environment
 sub set_environment {
     my $host = shift;
-    my $gkb = $hosts{$host};
+    my $gkb = $hosts{$host} // 'gkb';
```

Comment:
I'm not sure why "gkb" is a good default for a host. Isn't the host supposed to refer to the hostname of the machine?

Reply:
Ok.